### PR TITLE
Create ieg-europen-history-en.csl

### PR DESCRIPTION
**Pull request by MonikaBarget**
Leibniz Institute of European History

### CODE REVIEW

File: ieg-european-history-en.csl [hunk 1]
```diff
@@ -0,0 +1,367 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" et-al-min="4" et-al-use-first="1" et-al-subsequent-min="4" et-al-subsequent-use-first="1" page-range-format="expanded" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>IEG-2-en</title>
```
Comment (POBrien333) [inline ieg-european-history-en.csl:4]: Please input the full, written out title here.
`IEG-2` could go into title-short if necessary. Just below.

File: ieg-european-history-en.csl [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+    <choose>
+      <if is-numeric="volume">
+        <text variable="volume"/>
+      </if>
+      <else>
+        <text variable="volume"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="collection">
```
Comment (POBrien333) [inline ieg-european-history-en.csl:300]: This macro is not used and hence giving you the validation error. Delete.

File: ieg-european-history-en.csl [hunk 3]
```diff
@@ -0,0 +1,367 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" et-al-min="4" et-al-use-first="1" et-al-subsequent-min="4" et-al-subsequent-use-first="1" page-range-format="expanded" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>IEG-2-en</title>
+    <id>http://www.zotero.org/styles/ieg-2-en</id>
+    <link rel="self" href="http://www.zotero.org/styles/ieg-2-en"/>
+    <author>
```
Comment (POBrien333) [inline ieg-european-history-en.csl:7]: Please add the template style you used with label `rel="template`

File: ieg-european-history-en.csl [hunk 4]
```diff
@@ -0,0 +1,367 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" et-al-min="4" et-al-use-first="1" et-al-subsequent-min="4" et-al-subsequent-use-first="1" page-range-format="expanded" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>IEG-2-en</title>
+    <id>http://www.zotero.org/styles/ieg-2-en</id>
+    <link rel="self" href="http://www.zotero.org/styles/ieg-2-en"/>
```
Comment (POBrien333) [inline ieg-european-history-en.csl:6]: This is giving you a validation error. The style self link and ID both need to have the same name as the file name itself. Please match these up.

Comment (POBrien333) [inline ieg-european-history-en.csl:6]: Are the guidelines for this style publicly available?

**Comment by adam3smith** [comment]
@MonikaBarget please see the questions above

**Comment by adam3smith** [comment]
Closing for inactivity -- happy to reopen if you get back to this. Just leave a message here.

**Pull request closed without merging**